modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -19,7 +19,10 @@ class SpecialIssueUpdater
     # @param contention_id [Hash(claim_id:, code:, name:)] Identifier to match existing contention
     # @param special_issues [Array(String)] List of special issues to append
     # @param auto_claim_id [Integer] default: nil
-    def perform(user, contention_id, special_issues, auto_claim_id = nil)
+    def perform(contention_id, special_issues, auto_claim_id)
+      user = bgs_headers(auto_claim_id)
+      return if user.nil?
```

Comment:
we wouldn't want these returns here. a return without an error would tell sidekiq that everything seemed fine and we would end the job with an unsuccessful submission.

Reply:
I'd 2nd this with we should decide what to do (raise an exception/log/something) and write a test for it too.